packages/components/tsconfig.json

```diff
@@ -5,6 +5,7 @@
 		"types": [
 			"gutenberg-env",
 			"gutenberg-test-env",
+			"css-modules",
```

Comment:
In order to have types for the `style.module.css` everywhere, I moved the `declare module *.module.css` TypeScript definition to the global `typings` folder. This makes it available for the entire Gutenberg repo.